contracts/script/deploy/router/README.md

```diff
@@ -28,13 +28,15 @@ Create a configuration file in the `config/` directory with the following format
 ```json
 {
   "initialOwner": "0x0000000000000000000000000000000000000001",
-  "initialCertVerifier": "0x0000000000000000000000000000000000000002",
+  "initABNs": [0],
+  "initialCertVerifiers": ["0x0000000000000000000000000000000000000002"],
```

Comment:
no longer matches the config

Reply:
Actually, why is this even in the README? Can I just delete this part and just say to refer to the example_config?

Reply:
https://github.com/Layr-Labs/eigenda/pull/1787/commits/91082bb630bf183f4330c9ca5abbd3b2799f36d4